scripts/configure-multinode.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+set -e
+
+WORKER_NODE_HOSTNAME=${WORKER_NODE_HOSTNAME:-"worker-node"}
+: "${WORKER_NODE_IP:=$(getent hosts "${WORKER_NODE_HOSTNAME}" | awk '{print $1}')}"
+KUBELET_DIR=${KUBELET_DIR:-"/home/microshift/kubelet"}
+
+remote() {
+  if [ "$#" -lt 2 ]; then
+    echo "Usage: remote remote_host command [arg1 arg2 ...]"
+    return 1
+  fi
+
+  remote_host="$1"
+  shift
+
+  ssh -q -t -t "microshift@${remote_host}" "$(printf "%q " "$@")"
+}
+
+# pre-approve worker node host key
+ssh-keyscan "${WORKER_NODE_HOSTNAME}" >> ~/.ssh/known_hosts
```

Comment:
Should we consider adding `ssh-copy-id microshift@${WORKER_NODE_HOSTNAME}` to avoid entering the password during each ssh call?